src/modules/dashboard/components/ColonyHome/WrongNetworkDialog/WrongNetworkDialog.tsx

```diff
@@ -13,12 +14,15 @@ const MSG = defineMessages({
   },
   description: {
     id: 'dashboard.ColonyHome.WrongNetworkDialog.description',
-    defaultMessage: 'Please connect to the appriopriate Ethereum network.',
+    defaultMessage: 'Please connect to xDai chain. <a>Learn more.</a>',
```

Comment:
Again, this is not correct. What if we deploy this to mainnet. Then the user has the wrong network selected in the wallet. Then they receive e message saying: "Please connect to the xDai chain", while they are on ethereum mainnet.

See the problem ?

So this should be dynamic, just as everything. And this should use the same logic of detecting the network as `checkIfNetworkIsAllowed` _(but not the same util method)_

In this case, just take the current network where the dapp is deployed (from `process.env.NETWORK`, map it using the `NETWORK_DATA` constant to get it's "official" name _(the `name` prop)_ and then use that as a value in this message.

```suggestion
    defaultMessage: 'Please connect to {networkName}. <a>Learn more.</a>',
```